somleng_scfm_tools/upload_csv_contacts_to_scfm.py

```diff
@@ -0,0 +1,63 @@
+import json
+import csv
+import argparse
+import http.client
+
+from core_data_modules.logging import Logger
+
+log = Logger(__name__)
+log.set_project_name("UploadCsvContactsToScfm")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Uploads a phone number:metadata from a csv file with header "
+                                                 "to SCFM somleng")
+
+    parser.add_argument("scfm_instance_url", metavar="scfm-instance-url",
+                        help="Url to the server hosting the scfm instance in the format {scfm.somleng.org}")
+    parser.add_argument("scfm_contacts_api_key", metavar="scfm-contacts-api-key",
```

Comment:
I think the thing we'd usually do here is to upload this api key to the credentials bucket then pass in the standard gcloud credentials file here and a URL. Should we do that here too? Check with Luke if unsure.